apps/modeler/src/scene_tree.cpp

```diff
@@ -320,12 +310,10 @@ pcl::modeler::SceneTree::slotICPRegistration()
   ThreadController* thread_controller = new ThreadController();
 
   QList<RenderWindowItem*> selected_render_window_items = selectedRenderWindowItems();
-  for (QList<RenderWindowItem*>::iterator selected_render_window_items_it = selected_render_window_items.begin();
-    selected_render_window_items_it != selected_render_window_items.end();
-    ++ selected_render_window_items_it)
+  for (auto &selected_render_window_item : selected_render_window_items)
```

Comment:
In the previous two replacements in this file, no reference was added because it was a pointer container. Now we have once again a pointer container and a reference was included. It's not being consistent.

Reply:
There are more inconsistencies in this file. Let's pick something and stick with it. Personally I'm ok in retrieving a copy for containers of point. I'm not sure if we should adhere to the no brainer proposal from @taketwo already or if this is to be stalled until we reach a definitive conclusion in the use of `auto`.

Reply:
Currently I'm waiting for decision regarding `auto`.

Reply:
So the conclusion was that `auto` is up to you (better though if consistent), but please add reference whenever possible (i.e. in all cases where we don't actually want to work with a copy).